src/decision.js

```diff
@@ -0,0 +1,26 @@
+const docx = require('docx');
+const fs = require('file-system');
```

Comment:
You can use an `import` statement instead `require`

Reply:
Unfortunately node doesn't support `import` statement.